src/librustc_const_eval/diagnostics.rs

```diff
@@ -452,12 +452,14 @@ enum Method { GET, POST }
 
 
 E0297: r##"
+#### Note: this error code is no longer emitted by the compiler.
```

Comment:
This seems to be what we have done in the past. I see some instances of `##` and some with `####`. 

Personally, I long to get rid of `--explain XXX` and just have a "long error mode" in which the long explanations are emitted 'inline'. That would basically make this point irrelevant.